### PR TITLE
Merge document about operational data between FP and BizTalk2020

### PR DESCRIPTION
**Pull request by Elvis-Shi**
"Configure the Power BI operational data feed in BizTalk Server" is splitted into "Configure Operational data service for Feature pack" and "Operational Data Service"

### CODE REVIEW

File: biztalk/TOC.md [hunk 1]
```diff
@@ -480,7 +480,7 @@
 ###### [Keyboard Shortcuts Specific to Port Connectors](core/keyboard-shortcuts-specific-to-port-connectors.md)
 ## [Configure the feature pack](core/configure-the-feature-pack.md)
 ### [Track data to Azure](core/send-tracking-data-to-azure-application-insights-using-biztalk-server.md)
-### [Configure data feed for Power BI](core/configure-the-operational-data-feed-for-power-bi-with-biztalk-server.md)
+### [Configure Operataional Data Service](core/fp2016-configure-operational-data-service.md)
```
Comment (msnehagup) [inline biztalk/TOC.md]: Do we need to rename the title? Can we leave it as before?

File: biztalk/core/operational-data-service.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* Install IIS on the [!INCLUDE[btsBizTalkServerNoVersion_md](../includes/btsbiztalkservernoversion-md.md)]. 
+* [Configure the REST APIs](../install-and-config-guides/configure-biztalk-server.md) for **Operational data service**.
+* Optional. Install and configure a [Power BI Gateway](https://powerbi.microsoft.com/gateway/) to connect [PowerBI.com](https://powerbi.microsoft.com) with your on-premises BizTalk Server. If you're not using an on-premises BizTalk Server, then you don't need the gateway.
+
+
+## Use Power BI to Load Operational Data.
+
+1. Download and install the [Power BI Desktop](https://powerbi.microsoft.com/desktop/) on your BizTalk Server. You can select to open it, which is optional. If you have a work or school account, you may have access to Power BI. Try signing in with that account. Or, you can try it for free after signing up.
+2. Open the `\Program Files (x86)\Microsoft BizTalk Server 2016\OperationalDataService` folder, and open the `BizTalkOperationalData.pbit` file:
```
Comment (msnehagup) [inline biztalk/core/operational-data-service.md:51]: Should we remove '2016' from the folder path in the text, and the screenshot?

File: biztalk/core/operational-data-service.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Couldn't authenticate
+If you get `couldn't authenticate with the credentials provided` message similar to the following, then it’s possible your application pool identity doesn’t have enough access to the BizTalk Server databases. You can change the appPool identity within IIS to an account with more privileges, maybe your signed-in user account (which has local admin privileges).
+
+![Couldn't authenticate with the credentials provided](../core/media/operational-data-authentication-error.png)
+
+## Do more
+This is just the beginning. Power BI also has a gateway that can be installed on an on-premises BizTalk Server. Using the gateway, you can publish your dashboard, get real-time data, and create a schedule to refresh the dashboard. The following blog does a great job detailing these steps:
+
+* [How to publish BizTalk operational data on Power BI – Step-by-step configuration](https://blog.sandro-pereira.com/2017/05/07/biztalk-server-2016-feature-pack-1-how-to-publish-biztalk-operational-data-power-bi-step-by-step-configuration-part-3/)
```
Comment (msnehagup) [inline biztalk/core/operational-data-service.md:67]: this link is not for an official documentation - should it be included here?

File: biztalk/core/operational-data-service.md (hunk 2 above)
Comment (Elvis-Shi) [reply biztalk/core/operational-data-service.md:51]: Yes!

File: biztalk/core/operational-data-service.md (hunk 3 above)
Comment (Elvis-Shi) [reply biztalk/core/operational-data-service.md:67]: This was there, and i don't find any real reason to remove it, let's keep it.

Comment (SanjivGupta-SanGupt) [reply biztalk/core/operational-data-service.md:67]: We can make a general call on this topic - referencing external content within official Microsoft documents.
For restructuring. I am staying focused on the restructure aspect for the time being

**Review by SanjivGupta-SanGupt — APPROVED** [review]

**Pull request merged**